wekesa931/Questioner
# [Feature #164126139] Display all meetups

#### What does this PR do?
It fetches all meetups created by the admin and displays them on the user dashboard
#### Description of Task to be completed?
To have the user able to see all the meetups created by the admin user
#### How can this be tested
Go to the path
`
https://wekesa931.github.io/Questioner/UI/routes/user
`
Create a user account
Log in to your account
You should be able to see all the meetups
#### What are the relevant pivotal tracker stories?
        #164126139
![dashhmtp](https://user-images.githubusercontent.com/31995387/53130831-516fa980-357c-11e9-81be-629c0c1e2c96.png)
